react/src/pages/EndpointDetailPage.tsx

```diff
@@ -107,14 +111,8 @@ const EndpointDetailPage: React.FC<EndpointDetailPageProps> = () => {
   const [openChatModal, setOpenChatModal] = useState(false);
   const [currentUser] = useCurrentUserInfo();
   // const curProject = useCurrentProjectValue();
-  const [paginationState] = useState<{
-    current: number;
-    pageSize: number;
-  }>({
-    current: 1,
-    pageSize: 100,
-  });
-  const { message } = App.useApp();
+  const baiClient = useSuspendedBackendaiClient();
+  const navigate = useNavigate();
```

Comment:
Is there any specific reason to use `useNavigate()`? If not, please use `webuiNavigate` instead.

Reply:
I've changed the order of the declarations in this section, and haven't added or deleted anything new. I was using navigate in the existing code, but I'll change that as well.